### PR TITLE
Make download client categories uniform

### PR DESCRIPTION
**Pull request by l-t-k**
Use newznabs' defaults
http://newznab.readthedocs.org/en/latest/misc/api/

### CODE REVIEW

**Comment by l-t-k** [comment]
Fix for
https://github.com/Hellowlol/HTPC-Manager/issues/365

File: interfaces/default/js/newznab.js [hunk 1]
```diff
@@ -71,7 +71,54 @@ function search(query, catid, indexer) {
...
+			// 7020 = ebook
+			// 7030 = comics
+			if(item.attr['category'] == 7020) {
+			    var clean_category = "ebook"
+			}
+			else if(item.attr['category'] == 7030) {
+			    var clean_category = "comics"
+			}
+			else {
+			    var clean_category
```
Comment (Hellowlol) [inline interfaces/default/js/newznab.js]: You making this "undefined" here. why?

Comment (l-t-k) [inline interfaces/default/js/newznab.js]: Thanks for the feedback. I wanted to make clean_category equal to:
https://github.com/kooimens/HTPC-Manager/blob/patch-7/interfaces/default/js/newznab.js#L75
if not matches from the manual set categories, but I now understand I can just leave it like it is in de new commit:)

File: interfaces/default/js/newznab.js [hunk 2]
```diff
@@ -71,7 +71,48 @@ function search(query, catid, indexer) {
...
+			// 1 = console
+			// 2 = movies
+			// 3 = audio
+			// 4 = pc
+			// 5 = tv
+			// 6 = xxx
+			// 7 = other
+			var main_category = item.attr['category'].split("0",1)
+			if(main_category == 1) {
+			    var clean_category = "console"
```
Comment (Hellowlol) [inline interfaces/default/js/newznab.js]: Can you remove all the var from the lines you added? Its already defined so there no need to do it again.

Comment (l-t-k) [inline interfaces/default/js/newznab.js]: Right, I get it! Done.

**Comment by Hellowlol** [comment]
thank you.

**Pull request merged**